index.js

```diff
@@ -25,6 +25,8 @@ var DEFAULT_IGNORE_PATTERNS = [
 
 var DEFAULT_CONFIG = {
   configFile: path.join(__dirname, 'rc', '.eslintrc'),
+  reset: true,
```

Comment:
Isn't this not necessary anymore with eslint 1.0.0 since reset is the default behavior, i.e. there are no default rules? I just removed this in https://github.com/feross/standard/commit/4f7b20d621441e8ee2c60a375c112a56a3d64e75 ;)